lib/avalanche/token_cache.ex

```diff
@@ -26,29 +27,44 @@ defmodule Avalanche.TokenCache do
 
   @spec start_link(any()) :: Supervisor.on_start()
   def start_link(_) do
-    Mentat.start_link(name: @cache, ttl: @ttl)
+    import Cachex.Spec
```

Comment:
why do we import here?

Reply:
could we do a direct call in `Cachex.Spec.expiration/3` instead?

Reply:
I think it needs to be imported because it's a `defrecord`. I could do an `only: [:expiration, 2]`? it's only scoped to this function scope though

Reply:
It won't work if you call it directly

Reply:
neat, thanks for checking 👍